### DISCUSSION TITLE
Do you have a plan to add more machines to deal with this very long evaluation queue?

### DISCUSSION THREAD

**Discussion by merak0514** [cyberbotics/wrestling#45] [General]
Hello, thanks for your effort. But can you add some mechanisms like executing parallelly to mitigate the long waiting time?

<img width="271" alt="image" src="https://user-images.githubusercontent.com/35158310/221869788-426bce39-e570-493e-9e9e-0939adc0a11c.png">

**Comment by omichel** [comment]
Unfortunately, the bubble sort ranking algorithm cannot run in parallel.
This very long evaluation queue is exceptional today because we introduced the "programming" language in every `participant.json` file and we had to re-launch the evaluation of all robot controllers. Hopefully this will soon be complete and normal operations could resume.

Sorry for the inconvenience.

**Reply by merak0514** [reply]
Thank you. I think the bubble sort ranking is not that fair. Why not consider algorithms like ELO?

**Reply by omichel** [reply]
I agree ELO would be better, but it is also more difficult to implement as competitors would have to explicitly choose an opponent. But this is a very good suggestion. We will think more about it.
